_topic_maps/_topic_map.yml

```
@@ -312,6 +312,10 @@ Topics:
     File: serverless-ha
 - Name: Configuring kube-rbac-proxy resources for Eventing
   File: kube-rbac-proxy-eventing
+- Name: Using ContainerSource with OpenShift Service Mesh
```

For a follow-up: Consider one of the following:
```suggestion
- Name: Using a container source with OpenShift Service Mesh
```
^ Probably preferable, to match the "Using a sink binding..." style of the following topic.

Or (note no markup because heading):
```suggestion
- Name: Using a ContainerSource object with OpenShift Service Mesh
```
If changed, apply the styling to the other 2 similar instances (the `eventing/containersource-with-ossm.adoc` assembly and the `modules/configuring-containersource-with-ossm.adoc` procedure) to match.